src/components/common/LikesAndFollowers/index.tsx

```diff
@@ -56,6 +56,6 @@ const RightWrapper = styled.div`
 const Divider = styled.div`
   width: 0.1rem;
   height: 1.6rem;
-  background-color: var(--color-gray-dk);
+  background-color: var(—color-gray-dk);
```

Comment:
자동으로 바뀐 것 같은데 --로 해야 적용돼요!
```suggestion
  background-color: var(--color-gray-dk);
```